src/ir/auth_logic/ast.h

```diff
@@ -19,6 +19,7 @@
 #ifndef SRC_IR_AUTH_LOGIC_AST_H_
 #define SRC_IR_AUTH_LOGIC_AST_H_
 
+#include "src/ir/datalog/program.h"
```

Comment:
These need to be after the system headers by convention: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes